tests/test_middleware.py

```diff
@@ -124,3 +154,11 @@ def test_make_ip_list(self, capsys):
         expected = [ip_network('1.2.0.0/16'), ip_network('127.0.0.1')]
         assert make_ip_list(ip_addresses) == expected
         assert capsys.readouterr().out.startswith('Could not exclude supplied ip address')
+
+
+class TestMakeExcludedHeaders:
+    def test_make_excluded_headers(self):
+        exclusions = {'User-Agent': r'Somebot 2.0', 'Content-Length': r'^\d{,4}$'}
+        expected = {'Content-Length': re.compile('^\\d{,4}$', re.IGNORECASE),
```

Comment:
I feel like it's odd to use raw strings in the lines above but not here and below... Not a big deal, just stood out to me.